Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -751,6 +751,12 @@ inline RefPtr<PathOperation> BuilderConverter::convertPathOperation(BuilderState
             return nullptr;
         }
 
+        bool hasPosition = rayValue.positionX();
+        if (hasPosition) {
+            LengthPoint position { convertPositionComponentX(builderState, *rayValue.positionX()), convertPositionComponentY(builderState, *rayValue.positionY()) };
```

Comment:
...and convertPositionComponent can convert CSSValuePair to LengthPoint.

Reply:
Fixed.